playground/pspin/pspin_spherical_system.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import division
+import numpy as np
+
+from pele.potentials import MeanFieldPSpinSpherical
+from pele.systems import BaseSystem
+from pele.landscape import smooth_path
+from scipy.misc import factorial
+
+def normalize_spins(x):
+    x /= (np.linalg.norm(x)/np.sqrt(len(x)))
+    return x
+
+def spin_distance_1d(x1, x2):
+    x1 = normalize_spins(x1)
+    x2 = normalize_spins(x2)
+    return np.linalg.norm(x1 - x2)
+
+def spin_mindist_1d(x1, x2):
+    x1 = normalize_spins(x1)
+    x2 = normalize_spins(x2)
+    print np.linalg.norm(x1-x2)/np.sqrt(len(x1))
+    return np.linalg.norm(x1-x2), x1, x2
```

Comment:
Are there any symmetries we have to account for here?

1.  Overall rotational symmetry?  I don't think so.
2.  Spin labeling?  Again I think the random J's kill this.

It seems correct, but have a think about it.